Applications/Spire/Source/Ui/TableBody.cpp

```diff
@@ -1469,3 +1473,16 @@ void TableBody::on_widths_update(const ListModel<int>::Operation& operation) {
       }
     });
 }
+
+bool TableBody::on_key_press(QWidget& target, QKeyEvent& event) {
+  if(typeid(target) == typeid(ScrollBox)) {
```

Comment:
```typeid``` can't test polymorphic types, instead ```dynamic_cast``` handles them.

We'll have to meet this Friday to go over this, I think this might need an update to the table specifications, but so far it's good.